manifests/o/OpenWhisperSystems/Signal/6.12.0/OpenWhisperSystems.Signal.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: OpenWhisperSystems.Signal
+PackageVersion: 6.12.0
+MinimumOSVersion: 7.0.0.0
+InstallerType: nullsoft
+Scope: user
+InstallModes:
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```
Signal can only be installed on a 64-bit OS.

---

> Signal can only be installed on a 64-bit OS.

Yeah, I tried to test it on a 32-bit OS, and it clearly says it requires a 64-bit. The developer didn't specify the requirements, though. Thank you for your assistance.